swap_meet/clothing.py

```diff
@@ -1,2 +1,34 @@
-class Clothing:
-    pass
\ No newline at end of file
+from swap_meet.item import Item
+class Clothing(Item):
```

Comment:
Definitely need to import `Item` so that we can use it as the parent class to `Clothing`.